airflow_spark_on_k8s_job_builder/core.py

```diff
@@ -439,6 +442,49 @@ def set_secrets(self, conf: Dict[str, Union[str, int, float]]) -> "SparkK8sJobBu
         self.get_job_params()["executor"]["secrets"].update(conf)
         return self
 
+    def add_global_persistent_volume(
+            self,
+            volume_name: str,
+            claim_name: str,
+            mount_path: str,
+            readonly: bool = False,
+    ) -> "SparkK8sJobBuilder":
+        """Adds a global persistent volume mounted to the driver and all executors"""
+        if not self.get_job_params().get("volumes"):
```

Comment:
minor nit (in the interest of reducing code lines) do we need this safety valve for setting default empty array for volumes? Maybe I'm missing something, but think since you're safe accessing w default value in 
```
        existing_volumes = self.get_job_params().get("volumes", [])
```
.. and then overriding the full thing, I'd say we don't need these first 2 lines

If this is true, then same holds valid for `volumeMounts` bellow I believe